docbook/gnc-docbookx.dtd

```diff
@@ -195,6 +195,7 @@ https://www.w3.org/2003/entities/2007/w3centities-f.ent
 <!-- Fixme: In April 2012, the XML project was retired
   For more information, please explore the Attic [http://attic.apache.org/]. -->
 <!ENTITY url-xalan "https://xml.apache.org/"> <!-- XSLT and XQuery processor @ apache -->
+<!ENTITY url-fq "https://github.com/finance-quote/finance-quote">
```

Comment:
Alphabetical order inside of sections means after url-fints as line 169.

Reply:
done